test/Microsoft.AspNetCore.Razor.Tools.Test/project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "testRunner": "xunit",
+  "dependencies": {
+    "dotnet-test-xunit": "2.2.0-*",
+    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.1.0-*",
```

Comment:
This is problematic - Mvc's build depends on RazorTooling today and this produces a cycle. You'd have to move these tests in to Mvc if you want to reference this project here. The alternative is to change this to pin it to the RTM version (`1.0.0`) and we can tweak the script to ignore this dependency.